### PR TITLE
mediatomb, revive old bep file

### PR DESCRIPTION
**Pull request by Begasus**
ffmpeg, mp4 and spidermonkey have been updated since this (old) release, function changed since then, builds and installs ok for x86 (on x86_gcc2) but fails with upnp error 117 ...

### CODE REVIEW

File: net-misc/mediatomb/mediatomb-0.12.1.recipe [hunk 1]
```diff
@@ -1,31 +1,85 @@
+SUMMARY="Open source UPnP Media Server"
 DESCRIPTION="MediaTomb is an open source UPnP MediaServer"
 HOMEPAGE="http://mediatomb.cc/"
-SOURCE_URI="http://downloads.sourceforge.net/project/mediatomb/MediaTomb/0.12.1/mediatomb-0.12.1.tar.gz"
+COPYRIGHT="2005 Gena Batyan, Sergey 'Jin' Bostandzhyan
+	2006-2010 Gena Batyan, Sergey 'Jin' Bostandzhyan, Leonhard Wimmer"
```
Comment (alaviss) [inline net-misc/mediatomb/mediatomb-0.12.1.recipe]:
```diff
-COPYRIGHT="2005 Gena Batyan, Sergey 'Jin' Bostandzhyan
-	2006-2010 Gena Batyan, Sergey 'Jin' Bostandzhyan, Leonhard Wimmer"
+COPYRIGHT="2005-2010 Gena Batyan, Sergey 'Jin' Bostandzhyan, Leonhard Wimmer"
```

File: net-misc/mediatomb/mediatomb-0.12.1.recipe [hunk 2]
```diff
@@ -1,31 +1,85 @@
+SUMMARY="Open source UPnP Media Server"
 DESCRIPTION="MediaTomb is an open source UPnP MediaServer"
 HOMEPAGE="http://mediatomb.cc/"
```
Comment (alaviss) [inline net-misc/mediatomb/mediatomb-0.12.1.recipe]: The `HOMEPAGE` is currently down I think, got error 1001 from CloudFlare

File: net-misc/mediatomb/mediatomb-0.12.1.recipe [hunk 3]
```diff
@@ -1,31 +1,85 @@
...
-SOURCE_URI="http://downloads.sourceforge.net/project/mediatomb/MediaTomb/0.12.1/mediatomb-0.12.1.tar.gz"
+COPYRIGHT="2005 Gena Batyan, Sergey 'Jin' Bostandzhyan
+	2006-2010 Gena Batyan, Sergey 'Jin' Bostandzhyan, Leonhard Wimmer"
+LICENSE="GNU GPL v2"
 REVISION="1"
-STATUS_HAIKU="stable"
-DEPEND=""
-MESSAGE="This port only builds with gcc4. Use 'setgcc gcc4' before building."
-CHECKSUM_MD5="e927dd5dc52d3cfcebd8ca1af6f0d3c2"
+SOURCE_URI="http://downloads.sourceforge.net/project/mediatomb/MediaTomb/$portVersion/mediatomb-$portVersion.tar.gz"
```
Comment (alaviss) [inline net-misc/mediatomb/mediatomb-0.12.1.recipe]:
```diff
-SOURCE_URI="http://downloads.sourceforge.net/project/mediatomb/MediaTomb/$portVersion/mediatomb-$portVersion.tar.gz"
+SOURCE_URI="https://downloads.sourceforge.net/mediatomb/mediatomb-$portVersion.tar.gz"
```

File: net-misc/mediatomb/mediatomb-0.12.1.recipe (hunk 2 above)
Comment (Begasus) [reply net-misc/mediatomb/mediatomb-0.12.1.recipe]: Yes, seems like it's removed from DNS services?

Comment (alaviss) [reply net-misc/mediatomb/mediatomb-0.12.1.recipe]: Should we change it to

    https://sourceforge.net/projects/mediatomb/

instead? At least that link is still alive.

Comment (Begasus) [reply net-misc/mediatomb/mediatomb-0.12.1.recipe]: Changed :)

**Pull request merged**